File: android/testData/projects/migrateToAppCompat/app/src/main/java/com/example/google/migrate2appcompat/MainActivity.java
package com.example.google.migrate2appcompat;

import android.app.ActionBar;
import android.app.Activity;
import android.app.Fragment;
import android.app.FragmentManager;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;
import android.widget.ShareActionProvider;

public class MainActivity extends Activity {

    private ShareActionProvider mShareActionProvider;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ActionBar actionBar = getActionBar();
        FragmentManager manager = getFragmentManager();
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.main, menu);
        mShareActionProvider = (ShareActionProvider) menu.findItem(R.id.share).getActionProvider();
        return true;
    }

    public static class MyFragment extends Fragment {
        public MyFragment() {
        }

        @Override
        public void onAttach(Activity context) {
            super.onAttach(context);
        }
    }
}

File: project-system-integration-tests/testData/sdkindexapp/src/main/java/com/example/sdkindexapp/MainActivity.kt
package com.example.sdkindexapp

import android.app.Activity
import android.os.Bundle
import android.util.Log
import android.widget.TextView

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val label = TextView(this)
        label.setText("Hello Sdk Index World!")
        Log.i("MainActivity", "Hello Sdk Index World!")
        setContentView(label)
    }
}